src/create-parts-v2.js

```diff
@@ -127,6 +137,8 @@ const run = async argv => {
 
   const limit = limiter(concurrency)
 
+  // get list of urls to files to process and create cars from them
+  // using the limiter
```

Comment:
same, technically it’s just dynamo items. we should do some renames here to be `getItems` to make that a little clearer. the primary key name being “url” would be quite difficult to change but method names like this should be easy.